libs/entitlement/src/main/java/org/elasticsearch/entitlement/bootstrap/EntitlementBootstrap.java

```diff
@@ -22,15 +22,29 @@
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Function;
 
 public class EntitlementBootstrap {
 
+    public static Map<Path, Boolean> pluginData;
```

Comment:
Nit, could we encapsulate these in a record (eg Args) and have a static getter for it? Then we can just do one check on the args being set or not.

Reply:
Won't that require an export to server?

Reply:
No, I'm suggesting a record local to this class, just for holding all the input to bootstrap. Server would still pass each argument individually to bootstrap.

Reply:
> could we encapsulate these in a record (eg Args) and have a static getter for it?

I wanted to to that :) this is not exactly used as a map anywhere, so it makes more sense to have a record.

Reply:
Ah, I see what you are saying. Sure, I'll do that